features/device-store/org.wso2.carbon.device.mgt.enterprise.store.feature/src/main/resources/store/extensions/assets/deviceType/workflows/asset_create.js

```diff
@@ -0,0 +1,8 @@
+var log = new Log();
+var execute = function(req,res,session){
+	log.info('Executing the Device-Type asset create ');
+};
+
+var process = function(req,res,session){
```

Comment:
Acknowledge, this code was belongs to IoT server ES extension app which is deprecated after moving to UUF based devicemgt app. So unable to fix the formatting issue.